source/agora/node/Ledger.d

```diff
@@ -790,10 +822,20 @@ private KeyPair[] getRandomKeyPairs ()
 // Use a transaction with the type 'TxType.Freeze' to create a block and test UTXOSet.
```

Comment:
@bpfkorea/core unittests should be self-contained. We shouldn't pack every single feature test into a single unittest block. In fact, this block is commented as testing only freeze transactions and the UTXO set, but now it also tests Enrollments too.

In the future if any of these tests end up having to be changed because of some feature change, then it will become very complicated to edit and fix the tests.

Reply:
I made self-contained unitest.